atmo/settings.py

```diff
@@ -91,6 +91,7 @@
     # setup bootstrap action depends on it to autotune the cluster.
     'MASTER_INSTANCE_TYPE': 'c3.4xlarge',
     'SLAVE_INSTANCE_TYPE': 'c3.4xlarge',
+    'CORE_SPOT_BID': '0.84',
```

Comment:
If yes, wrapping this into a `config` call that allows us to set this via an environment variable (see other settings for an example) would be useful